lib/java/logging-service-opensearch/src/main/module-info.java

```diff
@@ -0,0 +1,15 @@
+module org.enso.logging.service.opensearch {
+  requires java.net.http;
+  requires scala.library;
+  requires ch.qos.logback.classic;
+  requires ch.qos.logback.core;
+  requires static org.enso.logging.service.logback;
+  requires org.enso.logging.service;
+  // For jsoniter_scala
+  requires org.enso.scala.wrapper;
+  requires org.slf4j;
+  requires static org.openide.util.lookup.RELEASE180;
+
+  provides org.enso.logging.service.logback.RemoteAppender with
```

Comment:
Someone will ask for `RemoteAppender` and obtain implementation of `OpenSearchAppender`.